### DISCUSSION TITLE
SVG can`t be displayed in Card components

### DISCUSSION THREAD

**Discussion by Nimbaland** [neuefische/questions#343] [Web] [answered]
Ich want to see im Yoga Pose SVG which is in a Image Folder in public and import that in my db.json and then displayed on my Card component.

I create a db.json with different id and an create in public a Image Folder with my Yoga Asanas SVG and import thant in my db.json but i get a error to displayed on my Card component.

This is my Error:

![Screenshot 2022-11-30 at 11 28 03](https://user-images.githubusercontent.com/113900605/204772450-3c01d107-acfc-48ac-b934-9159f7bf47f0.png)

How to connet my SVG with my Card component to display my Yoga Asana?

**Comment by ahohnsen** [comment]
Hey Nimba,

Instead of importing the `Image` folder, you can add the path to the respective image for every exercise to your `db.json`.
If you need help with the path, have a look at [this question](https://github.com/neuefische/questions/discussions/324).

**Comment by stefankorder** [comment, marked as answer]
Hey Nimba,
how Andrea said. And you have to fix the path at the objects inside your db.json like `/../../Images/...svg`.

To fix the Hydration-Error you need to call your Randomizer inside a useEffect:

![grafik](https://user-images.githubusercontent.com/77627144/204790864-4240e970-f342-41da-8bbd-c220742377c4.png)
